qcodes/utils/helpers.py

```diff
@@ -42,6 +58,7 @@ class NumpyJSONEncoder(json.JSONEncoder):
     def default(self, obj: Any) -> Any:
         """
         List of conversions that this encoder performs:
+
         * ``numpy.generic`` (all integer, floating, and other types) gets
         converted to its python equivalent using its ``item`` method (see
```

Comment:
For this to format correctly this line and all other continuations should be indented by 4 spaces